### ISSUE TITLE
PAINT annotations are outdated (2017-08-27)

### ISSUE THREAD

**Issue by cmungall** [geneontology/go-site#581]

**Comment by cmungall** [comment]
```
curl -L ftp://ftp.pantherdb.org/downloads/paint/presubmission/gene_association.paint_pombase.gaf.gz | gzip -dc | head
!Created on Fri Sep 29 12:04:53 2017.
!PANTHER version: v11.1.
!GO version: 2017-08-27.

```

**Comment by selewis** [comment]
One specific question Huaiyu. When you migrate annotations from one PANTHER release to another do  you retain the original date on which the curator made the annotation?

One thing we noticed was that every annotation appeared to have identical dates (and while Marc is fast I don't think he managed thousands of annotations in one day)

**Comment by selewis** [comment]
Two additional thoughts -
1. For loading through the pipeline it would be much, much easier to be loading from the PANTHER family GAFs (and not the reshuffled MOD GAFs). That way it would be much easier to detect when the information for a particular family had change.

2. It would also be useful if the file names of the GAF files included the PANTHER release number. It was (as I recall) also included in the contents for the GAF file, but that requires opening the file to look. Be easier to pick up on if the PANTHER release were a part of the GAF file name.

**Comment by pgaudet** [comment]
I suggest we schedule a call to hash all these out (and make action points). I'll send an email. 

Pascale

**Comment by huaiyumi** [comment]
We can provide the updated GAF files. However, I notice a problem in the GO database. See links below.
http://amigo.geneontology.org/amigo/gene_product/UniProtKB:Q8WWZ4
http://amigo.geneontology.org/amigo/gene_product/WB:WBGene00016258
It seems that some annotations are duplicated. Some are dated 2015. This is actually a question I have before we release an update. Do updated GAF files include all the IBAs, or just the IBAs that are not in GO database? Based on the old versions of GAFs in svn, it seems that they include all IBAs, but the release process seems that the old IBAs are not removed from the database. That is why I am holding off the update.

**Comment by selewis** [comment]
So we learned something today (March 29) talking to @huaiyumi which is that the date that now appears in the released PAINT GAF files is not the date the annotation was last touched by a curator, but is the release date (i.e. when @huaiyumi creates these monthly file sets). Now that we've got a corrected understanding of how the date is used here we can move on (nothing to be done, and we'll leave to later a discussion of tracing curation history).

Ball is now in @dougli1sqrd 's hands to handle loading PANTHER release only when running the pipeline. (and filter out anything coming indirectly via the MODs)

@vanaukenk - @dougli1sqrd has already added a filter for the ancient ones, but it needs to be expanded somewhat more.

**Comment by cmungall** [comment]
__Update on this ticket__

the scope of this ticket is upstream changes that need to be made in the PANTHER release pipeline, so I am assigning this to Huaiyu only. Additional filtering that happens in GO Central is in ticket #587.

**Comment by cmungall** [comment]
Note that a consequence of getting stale files from PANTHER is that gene symbols are not in sync

See:
https://github.com/geneontology/helpdesk/issues/112

@huaiyumi when do you think we can have the PANTHER pipeline produce up to date files?

**Comment by thomaspd** [comment]
We want to completely remove all old IBA annotations with every release.  Is this the process?

**Comment by cmungall** [comment]
@thomaspd yes

(note that the filtering is being discussed in #587. The topic of this ticket is version of GO used in PAINT exports from Panther)

**Comment by pgaudet** [comment]
@cmungall do you want to rename the ticket then ?

**Comment by thomaspd** [comment]
@huaiyumi thinks the removal of old IBAs may not be complete.

**Comment by huaiyumi** [comment]
The following link shows IBA annotations that are not in the current GAF file:
http://amigo.geneontology.org/amigo/gene_product/PomBase:SPAC694.02
This is related to a ticket https://github.com/geneontology/go-annotation/issues/1914

Also check out the link from my previous post: http://amigo.geneontology.org/amigo/gene_product/UniProtKB:Q8WWZ4
For example, the annotation of ABCA10 to GO:0042626 is duplicated with two different dates, 20150213 and 20170427.

**Comment by cmungall** [comment]
@pgaudet 
> do you want to rename the ticket then ?

I guess we should if people keep reverting to talking about the filter here 😃 

> The following link shows IBA annotations that are not in the current GAF file

@huaiyumi the ticket for discussing the filtering of IBAs from the MODs is here: #587 - it's still open, which is why you see IBAs that were previously ingested by the MODs and then released as part of their official GAF. Can we follow up on 587?

To restate what this ticket is about: 

when I opened the ticket on Mar 26 the header from PAINT files was like this:

```
curl -L ftp://ftp.pantherdb.org/downloads/paint/presubmission/gene_association.paint_pombase.gaf.gz | gzip -dc | head
!Created on Fri Sep 29 12:04:53 2017.
!PANTHER version: v11.1.
!GO version: 2017-08-27.
```

which was troubling, as both the creation date and GO version were from August.

this is improved, but the version of GO is still lagging (same command, run today):

```
!Created on Thu Apr  5 23:42:21 2018.
!PANTHER version: v.13.1.
!GO version: 2017-12-27.
```

can we try and sync these with the GO central pipeline as much as possible

**Comment by huaiyumi** [comment]
@cmungall Of course, we will try to stay in sync with the GO release schedule. Would you please advise us when is the release date, and what is the time window that we should upload the updated GAFs.

**Comment by cmungall** [comment]
I'll leave that to @kltm and @dougli1sqrd but I expect a few days before the end of the month?

But my main concern is the disconnect between the GAF creation date "Apr  5" and the version of the GO ontology "2017-12-27". I would recommend always getting the latest release version of the ontology.
